docs/content/how-to/altering-tables.md

```diff
@@ -413,6 +413,48 @@ Supported Type Conversions.
 
 {{< generated/column_type_conversion >}}
 
+## Changing Column Default Value
+
+The following SQL changes default value of column `col_a` to `0`.
+
+{{< tabs "change-column-default-value" >}}
+
+{{< tab "Flink" >}}
+
+```sql
+ALTER TABLE my_table SET (
+    'fields.col_a.default-value' = '0'
+);
+```
+
+{{< /tab >}}
+
+{{< tab "Spark3" >}}
+
+```sql
+-- alter column
+ALTER TABLE my_table ALTER COLUMN col_a SET DEFAULT 0;
+
+-- or set properties
+ALTER TABLE my_table SET TBLPROPERTIES (
+    'fields.col_a.default-value' = '0'
+);
+```
+
+{{< /tab >}}
+
+{{< tab "Trino" >}}
+
+```sql
+ALTER TABLE my_table SET PROPERTIES fields.col_a.default-value = '0';
```

Comment:
Have you tested this?

Reply:
I copied this part.

https://github.com/apache/incubator-paimon/blob/cfddb74dd34fc51ff8a76b46bf54a3ed8e597d5d/docs/content/how-to/altering-tables.md?plain=1#L55-L61

Reply:
I found that what I wrote was wrong, I will delete this part.